engine/src/test/java/org/pentaho/di/pan/delegates/PanTransformationDelegateTest.java

```diff
@@ -0,0 +1,204 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+package org.pentaho.di.pan.delegates;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.pentaho.di.cluster.SlaveServer;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.KettleLogStore;
+import org.pentaho.di.core.logging.LogChannel;
+import org.pentaho.di.core.logging.LogChannelInterface;
+import org.pentaho.di.core.logging.LogLevel;
+import org.pentaho.di.repository.Repository;
+import org.pentaho.di.trans.Trans;
+import org.pentaho.di.trans.TransExecutionConfiguration;
+import org.pentaho.di.trans.TransMeta;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+/**
+ * Unit tests for PanTransformationDelegate.
+ */
+public class PanTransformationDelegateTest {
+
+  @Mock
+  private TransMeta transMeta;
+
+  @Mock
+  private Trans trans;
+
+  @Mock
+  private Repository repository;
+
+  @Mock
+  private SlaveServer slaveServer;
+
+  @Mock
+  private TransExecutionConfiguration executionConfiguration;
+
+  private LogChannelInterface log;
```

Comment:
PanTransformationDelegateTest.java mock `LogChannelInterface log;` so you don't need `KettleLogStore.init();`

Reply:
fixed